[2022-09-19 22:07:40,638] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: dag2.divide manual__2022-09-19T16:37:33.102927+00:00 [queued]>
[2022-09-19 22:07:40,644] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: dag2.divide manual__2022-09-19T16:37:33.102927+00:00 [queued]>
[2022-09-19 22:07:40,644] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-09-19 22:07:40,645] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-09-19 22:07:40,645] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-09-19 22:07:40,655] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): divide> on 2022-09-19 16:37:33.102927+00:00
[2022-09-19 22:07:40,657] {standard_task_runner.py:52} INFO - Started process 4712 to run task
[2022-09-19 22:07:40,660] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag2', 'divide', 'manual__2022-09-19T16:37:33.102927+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmphvtwwc3h', '--error-file', '/tmp/tmp8yyc0hcw']
[2022-09-19 22:07:40,660] {standard_task_runner.py:80} INFO - Job 22: Subtask divide
[2022-09-19 22:07:40,694] {logging_mixin.py:109} INFO - Running <TaskInstance: dag2.divide manual__2022-09-19T16:37:33.102927+00:00 [running]> on host 0eef05c196e2
[2022-09-19 22:07:40,737] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag2
AIRFLOW_CTX_TASK_ID=divide
AIRFLOW_CTX_EXECUTION_DATE=2022-09-19T16:37:33.102927+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-19T16:37:33.102927+00:00
[2022-09-19 22:07:40,738] {python.py:175} INFO - Done. Returned value was: 0.5
[2022-09-19 22:07:40,749] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=divide, execution_date=20220919T163733, start_date=20220919T163740, end_date=20220919T163740
[2022-09-19 22:07:40,791] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-19 22:07:40,812] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
